helm-charts/seldon-core-operator/templates/webhook.yaml

```diff
@@ -34,9 +20,7 @@ webhooks:
   - v1
   - v1beta1
   clientConfig:
-{{- if not .Values.certManager.enabled }}
```

Comment:
Not sure why this keeps getting removed, we should update the script to ensure it's kept, but for now best to revert to keep as is